src/main/java/com/uid2/shared/auth/Role.java

```diff
@@ -10,13 +10,21 @@ public enum Role {
     OPERATOR,
     OPTOUT, // Used for operators to call optout service's /optout/refresh API
     OPTOUT_SERVICE, // UID2-912 - Specifically for optout service only
+    @Deprecated
     CLIENTKEY_ISSUER,
+    @Deprecated
     OPERATOR_MANAGER,
+    @Deprecated
     SECRET_MANAGER,
+    @Deprecated
     ADMINISTRATOR,
-    SHARING_PORTAL,
+    SHARING_PORTAL, // corresponds to custom okta scope 'uid2.admin.ss-portal'
     PRIVATE_SITE_REFRESHER, // UID2-575 - For cronjob to trigger private site data generated
-
+    ALL, // corresponds to 'developer' okta group
```

Comment:
ALL? Suggest a word that implies restricted operations 

DEFAULT/LIMITED access?

Reply:
hmm..probably DEFAULT is more intuitive than ALL. I'll update this